website/src/components/Changelog/Gateway.tsx

```diff
@@ -8,6 +8,7 @@ export default function Gateway() {
 
   return (
     <Entries href={href} arches={arches} title="Gateway">
+      {/*
       <Entry version="1.1.X" date={new Date("2024-XX-XX")}>
```

Comment:
Good ol JS happily creating valid Date objects with invalid date strings.

Reply:
A Modest Proposal: Generate this page from Rust :P